controllers/databasecluster_controller.go

```diff
@@ -130,6 +130,8 @@ timeout server 28800s
 	finalizerDeletePXCPVC           = "delete-pxc-pvc"
 	finalizerDeletePSMDBPVC         = "delete-psmdb-pvc"
 	finalizerDeletePGPVC            = "percona.com/delete-pvc"
+	finalizerDeletePXCSSL           = "delete-ssl"
```

Comment:
What about psmdb?

Reply:
I actually meant to put this PR as a draft, I'm still looking at PSMDB, sorry for that.

Reply:
We don't need any finalizer for PSMDB because the SSL secrets have an owner reference to the `PerconaServerMongoDB` CR so they get automatically deleted when the DB cluster is deleted.